pkg/replication/replication.go

```diff
@@ -108,16 +109,24 @@ func (c *Config) AddRule(opts Options) error {
 	if opts.Disable {
 		status = Disabled
 	}
-	tokens := strings.Split(opts.Arn, ":")
+	arnStr := opts.Arn
+	if opts.Arn == "" {
+		arnStr = c.ReplicationARN
+	}
+	tokens := strings.Split(arnStr, ":")
```

Comment:
can you verify the number of tokens expected as well? @poornas

Reply:
that's the next check